lua/nvim-treesitter.lua

```diff
@@ -2,13 +2,19 @@ local api = vim.api
 local parsers = require'nvim-treesitter.parsers'
 local install = require'nvim-treesitter.install'
 local locals = require'nvim-treesitter.locals'
+local highlight = require'nvim-treesitter.highlight'
 
 local M = {}
 
--- This function sets up everythin needed for a given language
+-- This function sets up everything needed for a given language
 -- this is the main interface through the plugin
 function M.setup(lang)
   if parsers.has_parser(lang) then
+    api.nvim_command(string.format([[
+      autocmd NvimTreesitter FileType %s lua require'nvim-treesitter.highlight'.setup()
+    ]],
+    lang
+    ))
```

Comment:
after coming back a few times on these lines, it kind of bothers me, could you extract the string so the call to nvim_command is easier to read ?